pipelines/SpeciesTreeFromBusco/nextflow.config

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+/** See the NOTICE file distributed with this work for additional information
+* regarding copyright ownership.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+includeConfig "$ENSEMBL_ROOT_DIR/ensembl-compara/pipelines/nextflow.config"
+
+params {
+    dir = "$HPS_HOME/Workspace/busco_tree/small_set"
+    busco_proteins = "$HPS_HOME/Workspace/busco_tree/vertebrata_odb10/refseq_db.faa"
+    anno_exe = "$HPS_HOME/Workspace/busco_tree/ensembl-anno/ensembl-anno.py"
+    enscode = "$HPS_HOME/Workspace/busco_tree/"
+    scriptsDir = "$ENSEMBL_ROOT_DIR/ensembl-compara/src/python/scripts"
+    results = "$HPS_HOME/Pipelines/SpeciesTreeRes"
+}
+
+process.executor = 'lsf'
```

Comment:
```suggestion
```
I included in the `main.nf` the specific resource for lsf in the process doing stuff and missing the label - it is best practice on codon to ensure that we limit the processes to the explicit resources to ensure we are not greedy, this also makes debugging a little simpler if a process does fail with an out of memory issue, and will prevent the pipeline silently failing without knowing which process was more memory greedy than anticipated. `label` should also be included in the `collateBusco` process when the process has something to do

Reply:
Because the use of the `label` in each process for each resource specifies the `process.executor = 'lsf'`, so it becomes redundant

Reply:
Is see! Thanks for the suggestions!